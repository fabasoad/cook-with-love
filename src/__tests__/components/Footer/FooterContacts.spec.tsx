/// <reference types="jest" />
import { shallow } from 'enzyme'
import * as React from 'react'
import FooterContacts from '../../../components/Footer/FooterContacts'

describe(FooterContacts.name, () => {
  test('should render FooterContacts correctly', () =>
    expect(shallow(<FooterContacts />)).toMatchSnapshot()
  )
})
