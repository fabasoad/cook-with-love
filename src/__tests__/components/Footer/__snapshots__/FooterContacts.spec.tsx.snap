// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContacts should render FooterContacts correctly 1`] = `
<div
  className="row contacts"
>
  <FooterLocation />
  <FooterEmail />
</div>
`;
